resources/js/pages/user/[user]/game-checklist.tsx

```diff
@@ -0,0 +1,98 @@
+import { router } from '@inertiajs/react';
+import { useTranslation } from 'react-i18next';
+import { route } from 'ziggy-js';
+
+import { BaseSelectAsync } from '@/common/components/+vendor/BaseSelectAsync';
+import { EmptyState } from '@/common/components/EmptyState';
+import { GamesListItem } from '@/common/components/GamesListItem';
+import { UserBreadcrumbs } from '@/common/components/UserBreadcrumbs';
+import { UserHeading } from '@/common/components/UserHeading';
+import { usePageProps } from '@/common/hooks/usePageProps';
+import { useUserSearchQuery } from '@/common/hooks/useUserSearchQuery';
+import { AppLayout } from '@/common/layouts/AppLayout';
+import type { AppPage } from '@/common/models';
+import { GameListSection } from '@/features/users/components/GameListSection/GameListSection';
+
+const UserGameChecklist: AppPage = () => {
+  const { player, groups } = usePageProps<App.Community.Data.GameChecklistPageProps>();
+
+  const { t } = useTranslation();
+
+  const query = useUserSearchQuery({ initialSearchTerm: player.displayName });
+
+  const handleUserChange = (newUser: string) => {
+    router.visit(
+      route('user.game-checklist', {
+        user: newUser,
+        _query: route().queryParams,
+      }),
+    );
+  };
+
+  return (
+    <>
+      <AppLayout.Main>
+        <div>
+          <UserBreadcrumbs t_currentPageLabel={t('Game Completion Checklist')} user={player} />
+          <UserHeading user={player}>{t('Game Completion Checklist')}</UserHeading>
+
+          <div className="form-grid-container mb-4">
```

Comment:
Minor CSS suggestion (also applies to _achievement-list.tsx_):
```tsx
<div className="mb-4 flex items-center gap-2">
```
This centers the label and field and adds just a bit of whitespace between them.